src/components/atoms/FormSearch/style.scss

```diff
@@ -0,0 +1,15 @@
+.form-search {
+    display: flex;
+    position: relative;
+    font-family: var(--main-title-font);
+    text-transform: none;
+    width: 100%;
+
+    input {
```

Comment:
use class and not html tags for styling since if I will want to change button to input type="submit" or type="button" it will break your layout

Reply:
Changed to class